src/pamiq_core/utils/reflection.py

```diff
@@ -0,0 +1,3 @@
+def get_class_module_path(cls: type) -> str:
+    """Get the module path of a class."""
```

Comment:
docstringに引数と戻り値の説明を記述してください。
```suggestion
    """Get the module path of a class.
    
    Args:
        cls: ...
    
    Returns:
         ...
    """
```

Reply:
修正しました！